src/Components/Carousel/Carousel.css

```diff
@@ -3,15 +3,44 @@
   display: flex;
   width: 100%;
   height: 100%;
+  align-content: center;
 }
 
 .main-img{
   width: 100%;
-  max-width: 750px;
-  max-height: 950px;
+  max-width: 400px;
+  min-width: 150px;
   height: auto;
-  align-content: center;
   z-index: 1;
+  margin-top: 30px;
+}
+
+@media only screen and (min-width: 1231px) {
+  .carousel-wrapper {
+    position: fixed;
+    display: grid;
+    grid-template-areas:
+      "main main main"
+      ".  carousel  .";
+    grid-template-rows: 60% 30%;
+    align-content: center;
+    justify-content: center;
+    gap: 70px;
+  }
+}
+
+
+@media only screen and (max-width: 1230px) {
+  .carousel-wrapper {
+    display: grid;
+    grid-template-areas:
+      "main main main"
+      ".  carousel  .";
+    grid-template-rows: 60% 30%;
+    align-content: center;
+    justify-content: center;
+    gap: 70px;
```

Comment:
Use a relative unit. If the resolution of the display is even slightly different this might break the UI.